### PR TITLE
Oncofusion support and CLI env var fix

### PR DESCRIPTION
**Pull request by raylim**

### CODE REVIEW

File: src/luna/common/utils.py [hunk 1]
```diff
@@ -253,9 +253,8 @@ def validate_params(given_params: dict, params_list: List[tuple]):
     d_params = {}
     for param, dtype in params_list:
         if given_params.get(param, None) is None:
-            raise RuntimeError(
```
Comment (raylim) [inline src/luna/common/utils.py:256]: Raising an error here is incorrect behavior if the parameter is optional.

Comment (raylim) [reply src/luna/common/utils.py:256]: (optional as declared by the click decorator)

File: tests/luna/common/test_utils.py [hunk 2]
```diff
@@ -60,23 +60,6 @@ def test_validate_params__casting():
     }
 
 
-def test_validate_params__missing_value():
```
Comment (raylim) [inline tests/luna/common/test_utils.py:63]: I removed the exception thrown in the validate params function, which broke this test.

Comment (raylim) [reply tests/luna/common/test_utils.py:63]: I don't think we need to test missing parameters as that's a feature of click. i.e. click should be able to enforce presence/absence of parameters.

File: src/luna/common/utils.py (hunk 1 above)
Comment (armaank) [reply src/luna/common/utils.py:256]: Gotcha, this makes sense

**Review by armaank — APPROVED** [review]
Great work!

**Pull request merged**